index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/Yemen_National_Youth_Council/CSS/main.css" />
+    <link rel="stylesheet" media="(min-width: 325px)" href="/Yemen_National_Youth_Council/CSS/mobile.css" />
+    <link rel="stylesheet" media="(min-width: 768px)" href="/Yemen_National_Youth_Council/CSS/desktop.css" />
+
+    <link rel="stylesheet" href="bootstrap-4.0.0-dist/css/bootstrap.min.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Inter:wght@300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Yemen National Youth Council</title>
+  </head>
+  <body>
+    <header>
+      <nav id="mobileNav">
+        <img id="menuBtn" src="images/mobile_menu.svg" alt="" />
+        <img id="ynyLogo" src="images/logo.svg" alt="Yemen national youth council logo" />
+      </nav>
+      <div id="menuPage" class="menuPage">
+        <div class="menuPageContent" id="menuPageContent">
+          <img class="menuX" src="./images/x.svg" alt="close" />
+          <ul class="menuPageList">
+            <li>
+              <a href="about.html">About</a>
+            </li>
+            <li>
+              <a href="#Scd_page">Program</a>
+            </li>
+            <li>
```

Comment:
Great design @GeekyHacks 🤝

- [ ] Kindly ensure your design is a close replica of the design given. At the moment your design is completely different from the design provided. Kindly take a look at the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and refactor your project to match it.